src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -28,6 +28,7 @@ interface ScopeEntry {
     val name: String
     val element: RsElement?
     val subst: Substitution get() = emptySubstitution
+    val isNotLazyOrInitialized: Boolean get() = true
```

Comment:
Maybe just `isInitialized` it enough? Non lazy things are initialized by default